src/joystick.cpp

```diff
@@ -77,14 +77,16 @@ QString Joystick::getVendorString() const
 {
     QString temp = QString();
 
-    if (controller != nullptr)
+#if SDL_VERSION_ATLEAST(2, 0, 6)
```

Comment:
Good that you removed it :)

Reply:
https://github.com/AntiMicroX/antimicrox/blob/master/CMakeLists.txt#L22   
Here minimal SDL version is `2.0.6`

Reply:
If there would such a need we could also bump it a little